plugins/generator-1.16.4/forge-1.16.4/procedures/block_is_tagged_in.java.ftl

```diff
@@ -0,0 +1,2 @@
+<#include "mcitems.ftl">
+(BlockTags.getCollection().getOrCreate(new ResourceLocation((${input$b}).toLowerCase(java.util.Locale.ENGLISH))).contains(${mappedBlockToBlockStateCode(input$a)}.getBlock()))
```

Comment:
I don't think getOrCreate exists anymore in tag collection.

Reply:
Do you know what replaces `getOrCreate`? If not, I'll search on the doc but after I'm done with all the errors...

Reply:
Check item is tagged procedure I ported for reference and change for block tags.

Reply:
Procedure block removed from PR.